package.json

```diff
@@ -20,10 +20,13 @@
     "express": "^4.17.0",
     "express-handlebars": "^5.2.0",
     "express-session": "^1.16.1",
+    "find-config": "^1.0.0",
     "handlebars": "^4.7.6",
     "moment": "^2.29.1",
     "multer": "^1.4.2",
+    "mysql": "^2.18.1",
```

Comment:
The mysql import is hopefully superfluous since sequelize uses mysql2 and any direct dependencies on mysql are dodgy because mysql2 is already in here so 1) that can be used to avoid yet another dependency and 2) It is all the better to go through the one mysql2 api since dodging around it through a mechanism other that the one that sequelize uses could sidestep any functionality that mysql2 uses to talk to sequelize..  So 1) unnecessay2 and 2) dodgy.  This dependency is best avoided.  I suggest removing it and making sure the project still works without it.  Then if something breaks it'll be because some code pulls it in and in THAT case, the code should be revised to work with mysql2 instead.